mozilla_vpn/views/active_subscriptions_table.view.lkml

```diff
@@ -41,7 +41,8 @@ view: +active_subscriptions_table {
   dimension: plan_interval_type {
     description: "Indicates the plan interval type (1 year, 6 month, 1 month, etc)"
     type: string
-    sql: CONCAT(${plan_interval_count},"_",  ${plan_interval});;
+    sql: CASE WHEN ${product_name} = "Mozilla VPN & Firefox Relay" THEN CONCAT("bundle","_", ${plan_interval})
+      ELSE CONCAT(${plan_interval_count},"_",  ${plan_interval}) END;;
```

Comment:
This is really good tip. Thanks! 
Yeah, the `plan_type` in `mozilla_vpn` and `relay` should eventually moved in their ETL instead of having the definition in th multiple files. I will add this to my future direction for the project.